COBOL Programming Course #1 - Getting Started/COBOL Programming Course #1 - Getting Started.md

```diff
@@ -1647,23 +1685,23 @@ zowe jobs view sfbi JOB00906 105
 
 where `JOB00906` and `105` are obtained from the previous commands.
```

Comment:
Maybe we can change JOB00906 to JOB09413 to match zowe-jobs-submit-ds-and-view-spool-output.png, 105 should be changed to 104 too.

There are also occurrences of JOB00906 and 105 on the few lines above.